### DISCUSSION TITLE
Error 1026

### DISCUSSION THREAD

**Discussion by citlaligm** [JKISoftware/Caraya#174] [Q&A]
I started to see this error when running on my RT target. This is a very simple assertion but it fails and I get the same issue on a bigger project. Because of this error, the log isn't generated. 
The file exists and can be written by lvuser user. 

It seems to fail on the Destroy Test Suite.vi but there is no error upstream. Any ideas? Thanks!

![image](https://github.com/JKISoftware/Caraya/assets/10260895/c1fbbbdb-5c31-4389-9cef-766fb925f7db)

System Info

![image](https://github.com/JKISoftware/Caraya/assets/10260895/0b42f4bc-3b07-40cf-932d-58cb7a007879)

**Comment by francois-normandin** [comment]
Can you copy the full error message and call chain of that error?

The error seems to originate in the Test Manager Engine and is being reported as a result of running the test suite. The Destroy Test Suite node returns the error from the engine as it awaits the test results.

Based on what I see from the call chain in your screenshot, the error is on this node:
![image](https://github.com/JKISoftware/Caraya/assets/11728548/528b372e-b194-4bb4-87aa-7e1bd17be2be)

Error 1026 is an invalid refnum, suggesting that the VI is not in memory at the moment of the call. The Test Manager Initialize method is potentially called in each of the VIs you show on the block diagram. Having the rest of the call chain would be a tip as to when this is actually happening in the suite.

**Comment by francois-normandin** [comment]
Sorry, I was looking at the probe window and didn`t see the other call chain.
The Spawn Process should be called in the definition of the suite, not the destruction. I think the error also happens in the Define Test Suite, but is not getting reported.

**Comment by francois-normandin** [comment]
I never gave a second thought to this error stream being discarded because it cannot error out unless the `Process Container` VI is not in memory, but since it is statically linked... it seemed an impossibility. (`Define Test Suite (SimpleText Report.vi)`)
![image](https://github.com/JKISoftware/Caraya/assets/11728548/f200fa8b-fc63-4a06-a2b6-07e0707ee4c5)

If you merge the error into the output, do you see the 1026 error as well?

If so, then we need to figure out why this VI is not in memory.
![image](https://github.com/JKISoftware/Caraya/assets/11728548/83682e76-2361-4a2d-b965-f532d7716f97)
